tsconfig.dev.json

```diff
@@ -4,6 +4,10 @@
 		"./automation/**/*.ts",
 		"./lib/**/*.ts",
 		"./tests/**/*.ts",
-		"./typings/**/*.ts"
+		"./typings/**/*.ts",
+		"./automation/**/*.js",
+		"./lib/**/*.js",
+		"./tests/**/*.js",
+		"./typings/**/*.js"
```

Comment:
It works, that's pretty sweet - it also made me realize a bug where tsconfig.js.json was inheriting from tsconfig.json instead of tsconfig.dev.json :)